cmd/pr-reminder/main.go

```diff
@@ -346,6 +355,19 @@ func findPrsForUsers(users map[string]user, ghClient prClient) map[string]user {
 	return users
 }
 
+// filterLabels filters out those labels from the PR we are not interested in
+// and returns only those that are included in the interestedLabels set
+func filterLabels(labels []github.Label, interestedLabels sets.String) []string {
```

Comment:
I think we should have a unit test for this function.

Reply:
yes good catch